app/src/main/java/edu/usf/sas/pal/muser/util/EventUtils.kt

```diff
@@ -35,11 +35,13 @@ object EventUtils {
      * @return UiAction object
```

Comment:
Please add `position` description to Javadoc. I would call it `seekPosition` too just to be clear.